earthdata_download/download.py

```diff
@@ -121,13 +121,21 @@ def data_url_from_entry(entry, **kwargs):
 
 
 def _download_file_https(url, target, username, password):
+    def _download(s, t, user_agent=False):
+        with s.get(url=url, stream=True) as r:        
+            r.raise_for_status()
+            r.raw.decode_content = True
+            with open(t, "wb") as t:
+                shutil.copyfileobj(r.raw, t) if not user_agent else t.write(r.content)
```

Comment:
Just a matter of taste - I prefer to use single-line if statements only when you capture the value returned, e.g. `a = 1 if condition else 2`. Here, we make two different function calls and having them on separate lines is perhaps more clear:

```suggestion
                if user_agent:
                    t.write(r.content)
                else:
                    shutil.copyfileobj(r.raw, t)
```

Reply:
changed